### ISSUE TITLE
Data FAIR efforst at TEAM Sprint June 2018

### ISSUE THREAD

**Issue by romulogoncalves** [NLeSC/ldm-sig#19]
The following projects for the TEAM Sprint June 2018 will have some focus on Data FAIR.
- https://github.com/NLeSC/TEAM2018/blob/master/june/interstellar.md
- https://github.com/NLeSC/TEAM2018/blob/master/june/fair.md
- https://github.com/NLeSC/TEAM2018/blob/master/june/ewatercycle.md

It would be important to coordinate these three groups working together. Fair team aims to develop a tool to validate that data meets FAIR principles, while the other two teams aim to create data which meets the FAIR principles -- so they are two use cases for the first.

**Comment by c-martinez** [comment]
Interesting 'tool' for assessing data FAIRness:

https://www.ands-nectar-rds.org.au/fair-tool

Basically a questionnaire, filled by hand -- subjective? So an automated tool to measure all of this would be really useful!

**Comment by c-martinez** [comment]
Automated FAIR metric checking tool would fit in the context of this action plan (part of NPOS)

https://github.com/FAIR-Data-EG/Action-Plan/issues

**Comment by c-martinez** [comment]
More on automatic checking of FAIR metrics: http://linkeddata.systems:3000/

**Comment by c-martinez** [comment]
Python clien for the link above:

https://github.com/wilkinsonlab/FAIR_Evaluator_Client

**Comment by MartineDeVos** [comment]
During the teamsprint we manually applied the FAIR metrics to the data sets we used in the eWatercycle I project. The report can be found [here](https://tinyurl.com/y8bxblam)
Furthermore, we also did a FAIR check of existing data repositories, i.e., "suppose we store our data in this repo, to what extent would that make our data FAIR".
The report can be found [here](https://tinyurl.com/ycag5jd8)

These reports will be used as deliverables in the European Open Science Cloud (EOSC) pilot project.
